hadoop-dist/src/main/compose/README.md

```diff
@@ -0,0 +1,48 @@
+<!--
```

Comment:
this readme will not get published to the website, either add details here:
https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/site/markdown/SingleCluster.md.vm

or create a heading in the end like it is there for ```Fully-Distributed Operation``` in this page & then add a new page for docker

Reply:
@ayushtkn I've updated it with a new page for docker. I don't know how to test if it will get published in the website.

Reply:
do a mvn clean site and it will build the website locally, and you can check that